source/sync/data-model/data-model-map.txt

```diff
@@ -24,6 +24,87 @@ refer to :ref:`Development Mode <development-mode>`.
 To learn more about how Device Sync uses these two schemas,
 refer to the :ref:`Sync Data Model Overview <sync-schema-overview>`.
 
+.. _database-and-collection-name-mapping:
+
+Databases, Collections, and Objects
+-----------------------------------
+
+When you configure Device Sync, you :ref:`specify the data source 
+<sync-config-data-source>` where you want to store data. This data source 
+may contain multiple databases, and each database may contain multiple 
+collections.
+
+The :ref:`App Services schema <schemas>` maps your Realm Database object 
+names to collections in databases in your Device Sync data source. The 
+``title`` field in an App Services schema maps to the object type name in 
+Realm Database. Because the ``title`` name determines the mapping between 
+client objects and the appropriate Atlas collection, this name must be
+unique among all schemas in your synced data source.
+
+The ``title`` does not need to match the collection name. 
+
+.. example::
+
+   Consider an app with a database named ``Pets``. It may contain multiple 
+   collections, such as ``Canine`` and ``Feline``. The App Services schema
+   for the ``Canine`` collection might resemble :ref:`the example below 
+   <sync-app-services-schema-dog-example>`, where the ``title`` field of 
+   the schema is ``Dog``. That would map a Realm Database object called 
+   ``Dog`` to the ``Canine`` collection in the ``Pets`` database.
+
+   You could not have another schema whose ``title`` was ``Dog`` in the 
+   same cluster. For example, you could not sync an object with a ``title`` 
+   of ``Dog`` to both a ``Debug`` and a ``Test`` database in the 
+   same cluster. If you want to sync the same object to different 
+   collections for app development purposes, you must use different 
+   data sources - one cluster for development and a different cluster
+   for production.
+
+.. _map-realm-to-atlas-with-development-mode:
+
+Mapping with Development Mode
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When you :ref:`enable Development Mode <enable-development-mode>` in your 
+Device Sync configuration, App Services can automatically create collections 
+and schemas for Realm Database objects that you sync. It creates those 
+collections in the database whose :ref:`name you specify 
+<sync-config-database-name>` when you enable Development Mode.
+
+With Development Mode enabled, Sync looks for a collection whose App Services 
+schema has a ``title`` that matches the name of your Realm Database object 
+type. This collection could be in any database in your linked data source. 
+It doesn't have to be in the database name you add when you configure 
```

Comment:
not sure why `the database name` is throwing me off (it reads like collections are in a database name). why not just `the database` or `the database you specify`?
```suggestion
It doesn't have to be in the database you add when you configure 
```